common/src/main/java/com/microsoft/identity/common/internal/broker/BrokerData.kt

```diff
@@ -61,12 +59,6 @@ data class BrokerData(val packageName : String,
     companion object {
         val TAG = BrokerData::class.simpleName
 
-        /**
-         * Determines if the debug brokers should be trusted or not.
-         * This should only be set to true only during testing.
-         */
-        val sShouldTrustDebugBrokers = BuildConfig.DEBUG
```

Comment:
we're moving away bit-by-bit from broker validator, so the variable here should be kept instead.

Reply:
(the way to think of it is that BrokerValidator should only contain "operations". BrokerData provide the source of data, and therefore we should be controlling what data to operate on at the source class).

Reply:
I found it's not that easy to remove the variable from BrokerValidator, I reverted changes around "sShouldTrustDebugBrokers".